app/models/shopify/import/product.rb

```diff
@@ -0,0 +1,6 @@
+module Shopify
+  module Import
+    class Product < Shopify::Import::Base
```

Comment:
I really don't think it's necessary to have a wrapper around shopify_api gem, which is already a wrapper around activeresource and is exactly what we need.